docs/buildstrategies.md

```diff
@@ -134,7 +133,13 @@ For each namespace where you want to use the strategy, you also need to apply th
 account to run the strategy. If the service account is named `pipeline` (default), you can use:
 
 ```sh
-kubectl apply -n <namespace> -f  samples/v1beta1/buildstrategy/multiarch-native-buildah/
```

Comment:
Can you add the cluster-role installation to the block where `kubectl apply -f samples/v1beta1/buildstrategy/multiarch-native-buildah/buildstrategy_multiarch_native_buildah_cr.yaml` is done.

In this block here, it should then only be an apply of the rolebinding to the namespace.